src/odbc/src/statement.cpp

```diff
@@ -344,19 +344,25 @@ SqlResult::Type Statement::InternalSetAttribute(int attr, void* value,
       break;
     }
 
+    case SQL_ROWSET_SIZE: {
+      SqlUlen val = reinterpret_cast<SqlUlen>(value);
+
+      LOG_DEBUG_MSG("SQL_ROWSET_SIZE: " << val);
+
+      rowsetSize = val;
+
+      ard->GetHeader().rowsetSize = rowsetSize;
```

Comment:
Would this require the ARD fields be restored later?

Reply:
No, this is an explicit call to SQLSetStmtAttr